sacred/messagequeue.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python
+# coding=utf-8
+import sys
+
+if sys.version_info[0] == 2:
+    from Queue import Queue
+else:
+    from queue import Queue, Empty
+
+
+class SacredMQ:
```

Comment:
Old style classes work, but they behave slightly differently and are discouraged. So this is only polish :wink: